src/components/SearchForm.jsx

```diff
@@ -101,9 +89,13 @@ export const FormikSearch = withFormik({
     radius: Yup.number().positive('Radius must be a number'),
     timeFrame: Yup.string(),
   }),
-  handleSubmit(values) {
-    {/* this is where we would run dispatch or make an AJAX call */}
-    console.log(values)
+  handleSubmit(values, props) {
+    props.dispatch(submit(values))
   }
 })(SearchForm)
 
+export default connect(
+  dispatch => ({ 
+    submit: bindActionCreators(searchTweets, dispatch),
+  }),
```

Comment:
Here is our problem. No matter how I structure this connection, I can't seem to get the searchTweets action creator passed as a function to the handleSubmit method. In handleSubmit it's just undefined. I'm not sure what I'm missing, any help? Once this is corrected, it's a matter of getting the reducer (../reducers/search/index.js) organized correctly and we should be good to go.